docs/plugins.rst

```diff
@@ -240,6 +240,35 @@ secret values into the plugin's configuration page, and hit **Save**. Users shou
 then be able to log in with their Google account when they click the log in page
 and select the option to log in with Google.
 
+Extension
```

Comment:
Can you add to the OAuth Login section above in the docs, an explanation of registration vs login with OAuth, and when Girder Users are created as part of this flow, and what is stored or not stored in the Girder User that distinguishes the OAuth from say a password based login?